core/src/main/java/io/grpc/internal/AutoConfiguredLoadBalancerFactory2.java

```diff
@@ -167,7 +167,7 @@ Status tryHandleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
             ResolvedAddresses.newBuilder()
                 .setAddresses(resolvedSelection.serverList)
                 .setAttributes(attributes)
-                .setLoadBalancingPolicyConfig(lbConfig)
+                .setLoadBalancingPolicyConfig(lbConfig != null ? lbConfig.getConfig() : null)
```

Comment:
Looks like there isn't a case where `selection.config` is an error. We may just change `PolicySelection.config` from `ConfigOrError` to `Object`.